.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,10 @@
-# Description
-<!-- Please include a summary of the change and a link to the JIRA ticket. Please add any additional motivation and context as needed -->
+# JIRA link
+<!-- insert a link to the JIRA ticket -->
```

Comment:
I understand @makslion  point but in the case where it is a community PR it would also be nice to see a link to an issue which was raised. Maybe a bit of rewording might be better.

```suggestion
# Issue link
<!-- insert a link to the JIRA ticket, GitHub issue or discussion thread -->
```

Reply:
I'm not against this template in any way, I'm just trying to ensure that our templates are ready for a big and happy community. An `Issue link` would be lovely (as well as a new and fancy PR template :tada: ) It already looks good to me and with that change, I would be the happiest person in my room

Reply:
I like this suggestion. 
I'll not update the template just yet, as I want to give people more time for feedback. 
But if there is no opposition to this suggestion, consider it included. :)